testapp/settings.py

```diff
@@ -1,5 +1,12 @@
+import os
 from pathlib import Path
 
+from dotenv import load_dotenv
+
+load_dotenv()  # take environment variables from .env.
```

Comment:
IMO this is too complicated for a testapp - instead we just set the envvars directly in the `ci.yml` workflow file and you can easily set them in your virtualenv for local development. See https://github.com/maykinmedia/zgw-consumers/blob/main/testapp/settings.py#L10